src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -25,6 +25,15 @@
 // clang-format off
 #if defined(KOKKOS_ENABLE_CUDA)
 #  if defined(KOKKOS_COMPILER_CLANG)
+
+// Some Thrust (or CUB to be more precise) versions use __shfl instead of
+// __shfl_sync which was removed in PTX ISA version 6.4, also see
+// https://bugs.llvm.org/show_bug.cgi?id=43505.
+// For some reason, the issue could only be observed with clang
```

Comment:
The issue is not observed with nvcc because the automatic enabling of `CUB_USE_COOPERATIVE_GROUPS` is guarded with `__CUDACC_VER_MAJOR__` check that is only defined in nvcc.
See [this](https://github.com/NVIDIA/cub/pull/164#issuecomment-495502027) comment.

Reply:
I decided to check the cub version instead.